src/content/developers/tutorials/erc-721-vyper-annotated-code/index.md

```diff
@@ -17,7 +17,7 @@ In contrast to that, ERC-721 tokens are designed for assets that are similar but
 cartoons](https://www.cryptokitties.co/)
 or titles to different pieces of real estate.
 
-In this article we will analyze [Ryuya Nakamura's ERC-721 contract](https://github.com/vyperlang/vyper/blob/master/examples/tokens/ERC721.vy).
+In this article we will analyze [Ryuya Nakamura's ERC-721 contract](https://github.com/vyperlang/vyper/blob/master/vyper/builtin_interfaces/ERC721.py).
```

Comment:
Little confused, the original link is working for me and appears to be the contract being referenced. Could we revert this? Or am I missing something?